GOVERNANCE.md

```diff
@@ -244,7 +277,153 @@ The current NumFOCUS Subcommittee consists of:
 - Jon Sedar
 - Thomas Wiecki
 
-### Institutional Partners and Funding
+## Core Contributors
+Core Contributors are those individuals with commit or managing rights to any of
+the repositories or services of the PyMC Project. However, the exact permissions
+of all Core Contributors may not be the same and depend on their team memberships.
+Therefore, Core Contributors may act as Recurrent Contributors depending
+on the task at hand.
+We believe this to be necessary given the current size and scope of
+the PyMC project. Contributors may be proficient in some areas of the
+project while being inexperienced in some others.
+
+### Core Contributor membership
+To become a Core Contributor, one must already be a Recurring Contributor.
+Core Contributors can nominate any Recurring Contributor to become a
+Core Contributor. For the nomination to go forward, it has to be
+ratified by the Steering Council and the BDFL.
+For a nomination to be rejected, clear reasoning behind the decision must be
+shared with the rest of the team. People whose nomination has been rejected can
+be nominated at any time again in the future, three months after the previous
+nomination at the earliest.
+
+### Current Core Contributors
+
+<!-- we could list them here or link to the pymc-devs organization, however,
+linking to the pymc-devs org would require the core team not being secret.
+I actually don't know if this is already the case, if it's possible even nor
+obviously how it would be done. I do believe it's important to be clear and
+explicit about who are core contributors, especially for people interested
+in joining the project to reach out, but also for organizations and meet-ups
+to ask for speakers.
```

Comment:
Just checked it: The **maximum visibility setting for a team is to be visible within the organization**.

Outsiders can still learn about a team's existence if it's mentioned, but **only org members** can mention or inspect a "visible" team.

Reply:
Thanks! That means that it's either listing here or the team of core contributors not being public